packages/folks.rb

```diff
@@ -24,21 +24,13 @@ class Folks < Package
   depends_on 'libxml2' # R
   depends_on 'readline' # R
 
-  def self.build
-    system "meson setup #{CREW_MESON_OPTIONS} \
-    -Dbluez_backend=false \
+  gnome
+
+  meson_options '-Dbluez_backend=false \
     -Ddocs=false \
     -Deds_backend=false \
     -Dinstalled_tests=false \
```

Comment:
Do we need `-Dinstalled_tests=false` if we have `-Dtests=false`?

Reply:
Looking at https://gitlab.gnome.org/GNOME/folks/-/blob/master/meson_options.txt?ref_type=heads I see:

```
# Test options
option('tests', type: 'boolean', value: true, description: 'Build tests programs')
option('installed_tests', type: 'boolean', value: false, description: 'Install test programs')
```